### ISSUE TITLE
"Basic info" on manage users page is semantically misordered

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4104]
## Area or Component
Manage users 

## Quick Summary
The "Basic information" header is coded as an h4 element, but there is no preceding h3 element in front of it so we skip from h2 to h4.

## Accessibility Violation
Heading levels should only increase by one: https://dequeuniversity.com/rules/axe/4.4/heading-order?application=AxeChrome

## Action Requested
Change the heading level to h3 and ensure all other headings on the page are also in order.

## Screenshots (optional)
<img width="688" alt="Screen Shot 2022-07-27 at 4 57 44 PM" src="https://user-images.githubusercontent.com/80282552/181392133-939239a7-638f-4b14-94e8-52b85ba8c09a.png">

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1647999156&range=17:17
